pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -119,7 +120,14 @@ func NewFit(plArgs runtime.Object, h framework.Handle, fts feature.Features) (fr
 	strategy := args.ScoringStrategy.Type
 	scorePlugin, exists := nodeResourceStrategyTypeMap[strategy]
 	if !exists {
-		return nil, fmt.Errorf("scoring strategy %s is not supported", strategy)
+		supportedScoringStrategyTypes := make([]config.ScoringStrategyType, 0, len(nodeResourceStrategyTypeMap))
```

Comment:
Would it be better to move these under `ValidateNodeResourcesFitArgs`?

Reply:
Yes.

Reply:
That makes sense. The check logic has been moved to `ValidateNodeResourcesFitArgs`.